tests/service/ProjectAnalysisTests.fs

```diff
@@ -729,22 +729,13 @@ let ``Test project2 all uses of all signature symbols`` () =
                  ("file1", ((22, 45), (22, 47))); ("file1", ((22, 50), (22, 52)))]);
                ("DUWithNormalFields", [("file1", ((3, 5), (3, 23)))]);
                ("DU1", [("file1", ((4, 6), (4, 9))); ("file1", ((8, 8), (8, 11)))]);
-               ("field Item1", [("file1", ((4, 6), (4, 9))); ("file1", ((8, 8), (8, 11)))]);
-               ("field Item2", [("file1", ((4, 6), (4, 9))); ("file1", ((8, 8), (8, 11)))]);
+               ("field Item1", []); ("field Item2", []);
```

Comment:
There seems a loss of information here, is that expected?

Reply:
Yes, it's expected and fixes a bug: as there was no item case for fields, union case items were used instead which lead to other places consider fields and cases equal. This particular test baseline expected to get case uses as field uses which was wrong, since there are no uses of these fields.